README.md

```diff
@@ -1,8 +1,7 @@
 # Statick Markdown Plugins
 
-| Service | Status |
-| ------- | ------ |
-| Build   | [![Travis-CI](https://api.travis-ci.org/sscpac/statick-md.svg?branch=master)](https://travis-ci.org/sscpac/statick-md/branches) |
+![Unit Tests](https://github.com/sscpac/statick-md/workflows/Unit%20Tests/badge.svg)
+[![Codecov](https://codecov.io/gh/xydesa/statick-md/branch/master/graph/badge.svg)](https://codecov.io/gh/xydesa/statick-md)
```

Comment:
Should this be `xydesa` or `sscpac` in the URL?

Reply:
I think you're right that it should be sscpac, but I'm not sure how to set it up like that on codecov. I'd propose leaving it like this for now, and changing it if we figure out how.

Reply:
It might just work, so changing it and hoping.